R/wflow_toc.R

```diff
@@ -48,7 +46,16 @@ wflow_toc <- function(ignore_nav_bar = TRUE, clipboard = TRUE, project = ".") {
   # Create table of contents ---------------------------------------------------
 
   s <- wflow_status(project = project)
-  rmd <- rownames(s$status)[s$status$published]
+  if (only_published) {
+    rmd <- rownames(s$status)[s$status$published]
+    if (length(rmd)==0) {
+      warning("No suitable content to be added to the TOC found.
+If you wish to include unpublished contents, consider setting `only_published = FALSE`.")
+    }
```

Comment:
I like the warning. However, could you please move the check/warning to after the code block that applies the argument `ignore_nav_bar`? Another way to have no output would be if all of your files were already listed in the navigation bar, so it would be nice for the warning to apply to this situation as well.

Reply:
thanks, makes sense. 

I added  reference to `ignore_nav_bar` in the revised warning. 

I also added a line that ensures an empty character vector is returned when toc would be empty. Without that `return(invisible(character()))`, the usual message "The table of content of your project is on the clipboard." would still be printed below the warnings, which would then be rather confusing.